gobblin-docs/user-guide/Hive-Registration.md

```diff
@@ -0,0 +1,70 @@
+Gobblin has the ability to register the ingested/compacted data in Hive. This allows registering data in Hive immediately after data is published at the destination, offering much lower latency compared to doing data ingestion and Hive registration separately.
```

Comment:
Would suggest adding the following the top of the file:

```
# Table of Contents

[TOC]
```

It will automatically generate and update a Table of Contents for the entire page